client/allocrunner/taskrunner/task_runner.go

```diff
@@ -464,9 +470,36 @@ MAIN:
 	tr.logger.Debug("task run loop exiting")
 }
 
-func (tr *TaskRunner) handleTaskExitResult(result *drivers.ExitResult) {
+// handleTaskExitResult handles the results returned by the task exiting. If
+// retryWait is true, the caller should attempt to wait on the task again since
```

Comment:
Hm I just left it since I didn't quite follow. The return value is telling the caller that the results of the Wait are likely not accurate and it should be retried.

Reply:
Oh I get it now. It's retrying the wait on the driver. Because the actual Wait() call wasn't changed I forgot what was actually being retried. I was reading `retryWait` as the backoff duration and getting confused.

This makes sense. Ignore me!